test/index.js

```diff
@@ -0,0 +1,42 @@
+var expect = require('chai').expect
+var fs = require('fs')
+var path = require('path')
+var moduleFile = '../mask.js'
+
+describe('mask.js', function () {
+  before(function(done){
+    fs.unlink(path.join(__dirname, '../customers.json'), function(){
+      done()
+    })
+  })
+  it('must work by matching the resuls in test/customers.json', function(done){
+    var mask = require(path.join(__dirname, moduleFile))
+    mask(function(error, maskedData){
+      expect(error).to.be.null
+      var maskedTestData = require('./customers.json')
+      expect(maskedData).to.deep.equal(maskedTestData)
+      done()
+    })
+  })
+  it('must have readFile ', function(done){
```

Comment:
These tests aren't great -- we want to test that the students are using `readFile` and `writeFile`. @vicfriedman linked to an example.